components/forecast/WeatherTab.tsx

```diff
@@ -52,14 +55,19 @@ export const WeatherTab: React.FC<WeatherTabProps> = ({zone, center_id, date}) =
 
   if (isLoading) {
     return (
-      <Center style={StyleSheet.absoluteFillObject}>
-        <ActivityIndicator />
+      <Center style={{flex: 1}}>
+        <VStack space={8} style={{flex: 1}} alignItems={'center'}>
+          <Body>Loading current weather forecast for the {zone.name} zone...</Body>
```

Comment:
does this text really add anything that the activity indicator doesn't tell you? good question for design